src/lib/utils/format-branch-name.ts

```diff
@@ -0,0 +1,16 @@
+import { env } from "../../env.js";
+import { hyphenatedSummary } from "./hyphenated-summary.js";
+
+export const formatBranchName = (
```

Comment:
Might be nice to add an example of the config to either the example config or the readme